File: app/src/main/java/com/example/narthana/genpass/PreBuiltWordDBHelper.java
package com.example.narthana.genpass;

import android.content.Context;

import com.readystatesoftware.sqliteasset.SQLiteAssetHelper;

/**
 * Created by narthana on 26/10/16.
 */

public class PreBuiltWordDBHelper extends SQLiteAssetHelper
{
    private static final String DATABASE_NAME = "words.db";
    private static final int DATABASE_VERSION = 1;

    public PreBuiltWordDBHelper(Context context)
    { super(context, DATABASE_NAME, null, DATABASE_VERSION); }
}

File: app/src/main/java/com/example/android/genpass/data/PreBuiltWordDBHelper.kt
package com.example.android.genpass.data

import android.content.Context
import com.readystatesoftware.sqliteasset.SQLiteAssetHelper

/**
 * Created by narthana on 26/10/16.
 */

class PreBuiltWordDBHelper(context: Context):
        SQLiteAssetHelper(context, DATABASE_NAME, null, DATABASE_VERSION) {
    companion object {
        private const val DATABASE_NAME = "words.db"
        private const val DATABASE_VERSION = 1
    }
}